content/en/serverless/azure_functions/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Tracing Azure Functions
+---
+
+## Overview
+Azure Functions
+
+<!-- TODO: some more preamble -->
+
+This page explains how to collect traces from your Azure Functions. To collect metrics, install the [Datadog Azure integration][6].
+
+## Setup
+
+{{< programming-lang-wrapper langs="nodejs,python" >}}
+{{< programming-lang lang="nodejs" >}}
+1. **Install dependencies**. Run the following commands:
+   ```shell
+   npm install @datadog/serverless-compat
+   npm install dd-trace
+   ```
+
+   If your Azure Functions use a version of Node.js earlier than v18, ensure that you use a [compatible version of `dd-trace`][1].
+
+   To use [automatic instrumentation][2], you must use `dd-trace` v4.48+ (v4 release line) or v5.25+ (v5 release line).
```

Comment:
In the last week the v4 tracer release line status was updated to end of life. I think we can remove any references to the v4 release line and using versions of Node.js earlier than 18and instead just mention using v5.25+ for auto instrumentation.